src/test/java/de/retest/ui/descriptors/DefaultRetestIdProviderTest.java

```diff
@@ -118,12 +117,12 @@ public void already_known_ids_should_get_a_unique_suffix() {
 		when( identifyingAttributes0.get( "text" ) ).thenReturn( "+" );
 		when( identifyingAttributes0.get( "type" ) ).thenReturn( "+" );
 
-		assertThat( cut.getRetestId( identifyingAttributes0 ) ).isEqualTo( "retestid" );
+		assertThat( cut.getRetestId( identifyingAttributes0 ) ).isNotEmpty();
 
 		final IdentifyingAttributes identifyingAttributes1 = mock( IdentifyingAttributes.class );
 		when( identifyingAttributes1.get( "text" ) ).thenReturn( "+" );
 		when( identifyingAttributes1.get( "type" ) ).thenReturn( "+" );
 
-		assertThat( cut.getRetestId( identifyingAttributes1 ) ).startsWith( "retestid-" );
+		assertThat( cut.getRetestId( identifyingAttributes1 ) ).isNotEmpty();
```

Comment:
The test has no functionality, since the + in the type and text is cleaned up and a suffix is ​​created for both attributes. The test would be more useful if both attributes have the same text or type (which are not cleaned up) so that a suffix is ​​created for the second attribute.
The last assert could look like: assertThat( ...).isNotEqualTo( [Text or type of both] );

Reply:
Good catch!